lib/templates.ts

```diff
@@ -1454,3 +1476,40 @@ export const checkIfClosed = async (formId: string) => {
     return null;
   }
 };
+
+export const updateNotificationsSetting = async (
```

Comment:
I'm wondering if this function belongs in the `notifications` lib intead of the `templates` lib?